app/views/articles/index.html.erb

```diff
@@ -38,7 +38,7 @@
     <%# BEGIN Feed menu bar %>
     <main class="articles-list crayons-layout__content" id="articles-list" role="main">
       <h1 class="visually-hidden-header">Articles</h1>
-      <%= render(partial: "onboardings/task_card") if user_signed_in? %>
+      <%= render(partial: "onboardings/task_card") if user_signed_in? && current_user.saw_onboarding %>
```

Comment:
This check will be important for the creator onboarding flow; without it, every Forem creator will see that onboarding task card, which will not make sense/will not apply to them.

Reply:
**suggestion (nitpick, non-blocking):** Boolean columns automatically get predicate methods, should we use it here?

```suggestion
      <%= render(partial: "onboardings/task_card") if user_signed_in? && current_user.saw_onboarding? %>
```

Reply:
Oh, absolutely! Thanks for flagging that, it's much nicer. Latest commit includes this change 👍